src/main/scala/ru/otus/sc/game/model/GameMap.scala

```diff
@@ -0,0 +1,70 @@
+package ru.otus.sc.game.model
+
+import scala.util.Random
+
+object GameMap {
+  val size: Int      = 10
```

Comment:
Обычные константы можно обозначить синтаксически
```suggestion
  val Size: Int      = 10
```